client/layout/masterbar/masterbar-cart-button-style.scss

```diff
@@ -0,0 +1,28 @@
+.masterbar-cart-button {
+	display: flex;
+}
+
+.masterbar-cart-button button {
+	border-radius: 2px;
+	border: 1px solid var( --studio-gray-50 );
```

Comment:
Seems like we should get rid of the border on mobile:

![](https://cldup.com/odl-rm5ZfI.png)

We can also ditch the border radius on mobile FWIW.

Reply:
I think the border needs to disappear for all widths actually. It was probably there from a previous version of masterbar buttons.